phalcon/Mvc/Collection.zep

```diff
@@ -558,13 +560,13 @@ abstract class Collection implements EntityInterface, CollectionInterface, Injec
      * }
      * ```
      */
-    public static function findById(var id) -> <CollectionInterface> | null
+    public static function findById(var id) -> <CollectionInterface> | bool
```

Comment:
Could you please return this and all other similar changes (null -> bool) back.

Reply:
the "findFirst" and "findById" must return a null? Because in the original code, the return type was "Collection" or "false" ?

Reply:
Please make it consistent with https://github.com/phalcon/cphalcon/issues/14044

Reply:
Do the methods have to lift an exception?

Reply:
The `findFirst` etc. have been changed to return `null` or the record itself. In this case it will be a Collection object. Have a look at the implementation that the model has and let us replicate that. 

These are the changes made to make the methods return `null`

https://github.com/phalcon/cphalcon/pull/14089/files

Reply:
Normally these changes are made now